sktm/reporter.py

```diff
@@ -246,7 +248,27 @@ def __create_data(self, merge_report, result_set_list=[]):
                                                       test_result_dir,
                                                       index)
 
-            full_report += '\n' + build_report + run_report
+            console_result = next((test_result for test_result in test_run
+                                   if test_result.endswith(
+                                           'console_check.result'
+                                   )), None)
```

Comment:
This seems really neat, but could you please add a brief comment explaining what you're doing here and in cases above? Or perhaps rewrite those so they're more obvious? Thanks :)

Reply:
I'll add a separate commit with explanation